packages/kit/src/runtime/server/cookie.js

```diff
@@ -113,6 +116,14 @@ export function get_cookies(request, url, trailing_slash) {
 		 * @param {import('./page/types.js').Cookie['options']} options
 		 */
 		set(name, value, options) {
+			// TODO: remove this check in 3.0
+			const illegal_characters = name.match(INVALID_COOKIE_CHARACTER_REGEX);
+			if (illegal_characters) {
```

Comment:
Should we only warn once per cookie name? Else you might get flooded depending on how often you use them

Reply:
I dunno. We really don't want folks to miss this or their apps will break. If it's just one line in a thousand that their app logs then they might overlook it. I don't think this is terrible. Most people won't be affected, but we really do want folks to see it when they are